docs/dvl/dvl-protocol.md

```diff
@@ -182,7 +182,7 @@ Dead reckoning can be reset by issuing the `reset_dead_reckoning` command:
 {"command": "reset_dead_reckoning"}
 ```
 
-The response will be as follows if the reset is successful. If unsuccessful, `success` will be `false`, and a non-empty `error_message` will be provided.
+The response will be as follows if the message is received and flag is successfully set. If unsuccessful, `success` will be `false`, and a non-empty `error_message` will be provided.
```

Comment:
Is it important with the "note" below? If not I would write it more like this:

If the request is successfully received the response will have `success` set to `true`. The dead reckoning will have a delay of approximately 50ms until the positioning values being zeroed out. If the response is unsuccessful, the `success` will be `false` and a non-empty describing text will be returned in  `error_message`.

Reply:
[4cc23a5](https://github.com/waterlinked/waterlinked.github.io/pull/30/commits/4cc23a58715602dcbfbcc5c73f60f5808ebf526f)